.cfnlintrc

```diff
@@ -1,8 +1,10 @@
 templates:
 - ./**/*.yml
-#- ./**/*.yaml
+- ./**/*.yaml
 ignore_templates:
 - ./**/buildspec.yml
 - ./**/buildspec.yaml
 - ./**/buildspec*.yml
 - ./**/buildspec*.yaml
+ignore_checks:
+- W3045 # Ignoring this check because 'AccessControl' should still be supported for our use case
```

Comment:
We are using a param which linter thinks we should upgrade to a more granular one. It should not be a problem using the one we already have as long as we dont need more granular access control on the ones that issue this warning